components/odh-notebook-controller/controllers/notebook_oauth.go

```diff
@@ -209,6 +209,26 @@ func NewNotebookOAuthSecret(notebook *nbv1.Notebook) *corev1.Secret {
 	}
 }
 
+// NewNotebookOAuthClientSecret defines the desired OAuth client secret object
+func NewNotebookOAuthClientSecret(notebook *nbv1.Notebook) *corev1.Secret {
```

Comment:
As we would not be utilizing, opendatahub-operator secretgenerator, 
lets change the logic here, and write our own secret create

please take this as reference: https://github.com/opendatahub-io/kubeflow/blob/30afc1574e05bb05b21565db64e3b0ba5d6b9060/components/odh-notebook-controller/controllers/notebook_oauth.go#L190

and adjust this function , and for secret generation, 
utilize the logic of random function from here: https://github.com/opendatahub-io/opendatahub-operator/blob/c1671ab5fd11baea814f8acdee1bc448d502fb1c/controllers/secretgenerator/secret.go#L91
```suggestion
func NewNotebookOAuthClientSecret(notebook *nbv1.Notebook) *corev1.Secret {
  // Generate the client secret for the OAuth proxy
	randomValue := make([]byte, 32)
		for i := 0; i < secret.Complexity; i++ {
			num, err := rand.Int(rand.Reader, big.NewInt(int64(len(letterRunes))))
			if err != nil {
				return err
			}
			randomValue[i] = letterRunes[num.Int64()]
		}
		
	// Create a Kubernetes secret to store the cookie secret
	return &corev1.Secret{
		ObjectMeta: metav1.ObjectMeta{
			Name:      notebook.Name + "-oauth-client",
			Namespace: notebook.Namespace,
			Labels: map[string]string{
				"notebook-name": notebook.Name,
			},
		},
		StringData: map[string]string{
			"secret": string(randomValue),
		},
	}
```

and adjust the oauth-proxy to directly pick value from this secret